backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-plugins.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-plugins_{context}"]
+= OADP plugins
+include::_attributes/common-attributes.adoc[]
+:context: oadp-features-plugins
+:_content-type: CONCEPT
+
+toc::[]
+
+The OpenShift API for Data Protection (OADP) provides default Velero plugins that are integrated with storage providers to support backup and snapshot operations. You can create link:https://{velero-domain}/docs/v{velero-version}/custom-plugins/[custom plugins] based on the Velero plugins.
+
+OADP also provides plugins for {product-title} resource backups, OpenShift Virtualization resource backups, and Container Storage Interface (CSI) snapshots.
+
+[cols="3", options="header"]
+.OADP plugins
+|===
+|OADP plugin |Function |Storage location
+
+.2+|`aws` |Backs up and restores Kubernetes objects. |AWS S3
+|Backs up and restores volumes with snapshots. |AWS EBS
+
+.2+|`azure` |Backs up and restores Kubernetes objects. |Microsoft Azure Blob storage
+|Backs up and restores volumes with snapshots. |Microsoft Azure Managed Disks
+
+.2+|`gcp` |Backs up and restores Kubernetes objects. |Google Cloud Storage
+|Backs up and restores volumes with snapshots. |Google Compute Engine Disks
+
+|`openshift` |Backs up and restores {product-title} resources. ^[1]^ |Object store
+
+|`kubevirt` |Backs up and restores OpenShift Virtualization resources. ^[2]^ |Object store
+
+|`csi` |Backs up and restores volumes with CSI snapshots. ^[3]^ |Cloud storage that supports CSI snapshots
+|===
```

Comment:
Your call to include in this review or update.. 1.2.0 added the vsm plugin. Ah found jira OADP-2205

Reply:
Note: vsm is 1.2.0 only, and unfortunately will most likely be ripped out in 1.3.0. So perhaps just throw an asterisk on it :)